packages/psss/src/routes/PageRoutes.js

```diff
@@ -13,21 +15,25 @@ import { PrivateRoute } from './PrivateRoute';
 import { UnauthorisedView } from '../views/UnauthorisedView';
 import { NotFoundView } from '../views/NotFoundView';
 import { checkIsAuthorisedForCountry, checkIsAuthorisedForMultiCountry } from '../utils/auth';
+import { checkIsMultiCountryUser } from '../store';
 
-export const PageRoutes = React.memo(() => (
+export const PageRoutesComponent = React.memo(({ canViewMultipleCountries }) => (
   <Switch>
     <PrivateRoute exact path="/" authCheck={checkIsAuthorisedForMultiCountry}>
       <CountriesReportsView />
     </PrivateRoute>
     <PrivateRoute path="/weekly-reports/:countryCode" authCheck={checkIsAuthorisedForCountry}>
       <CountryReportsView />
     </PrivateRoute>
-    <PrivateRoute path="/alerts/:countryCode" authCheck={checkIsAuthorisedForCountry}>
```

Comment:
I don't know how I missed this the first time, but `alerts/outbreaks` and `alerts/archive` were going down this route and failing the auth check.

Reply:
Oops, I missed that too!